src/test/rustdoc/assoc-consts.rs

```diff
@@ -1,8 +1,8 @@
 pub trait Foo {
     // @has assoc_consts/trait.Foo.html '//*[@class="rust trait"]' \
-    //      'const FOO: usize;'
+    //      'const FOO: usize = 13usize;'
     // @has - '//*[@id="associatedconstant.FOO"]' 'const FOO: usize'
```

Comment:
Shouldn't this be:

```suggestion
    // @has - '//*[@id="associatedconstant.FOO"]' 'const FOO: usize = 13;'
```

?